src/appshell/qml/HomePage/HomeMenu.qml

```diff
@@ -120,7 +137,23 @@ Item {
                     radioButtonList.currentIndex = index
                     root.selected(modelData["name"])
                 }
+
+                onWidthChanged: {
+                    if (width < root.toggleWidth) {
+                        textShowed = ""
+                    } else {
+                        textShowed = title
+                    }
+                }
             }
         }
     }
+
+    onWidthChanged: {
+        if (width < root.toggleWidth) {
+            layout.accountInfoButton.showAvatarOnly()
```

Comment:
I think you should just correct root width instead of showing/hiding text for menu item
```
if (width < root.toggleWidth) {
    width = minWidth
} else {
    width = maxWidth
}
```

Reply:
I tested it and find the userName is still visible after `width = minWidth` , and it seems change `accountInfoButton`'s tilte is the simplest way to solve thath?